docs/source/DescriptionAndFeatures.rst

```diff
@@ -12,15 +12,15 @@ This Adder computes the sum of the inputs x1 and x2 and stores the result in the
 are available:
 
     1. *Cost-aware synthesis*:
-    In this synthesis scheme, additional circuit lines (representing circuit signals) are introduced to store the corresponding intermediate results of each operation—thereby providing the liberty to accordingly modify the circuit without having any effect on the original inputs (for more details, refer :cite:p:`wille2010syrec` and :cite:p:`wille2016syrec`). This results in a circuit description where the gate costs are kept moderate at the expense of a substantially larger number of additionally needed circuit lines. The following figure depicts the circuit which results when synthesizing the program from above.
+    In this synthesis scheme, additional circuit lines (representing circuit signals) are introduced to store the corresponding intermediate results of each operation—thereby providing the liberty to accordingly modify the circuit without having any effect on the original inputs. This results in a circuit description where the gate costs are kept moderate at the expense of a substantially larger number of additionally needed circuit lines. The following figure depicts the circuit which results when synthesizing the program from above.
 
     .. image:: images/costAwareAdder.svg
        :width: 400
        :alt: Adder circuit resulting from cost-aware synthesis.
        :align: center
 
     2. *Line-aware synthesis*:
-    This synthesis scheme aims to the keep the number of additional circuit lines as small as possible. To achieve this, the corresponding intermediate results of each operation are computed and stored using one of the available circuit lines (rather than additional circuit lines). Afterwards, the inputs are re-computed with the help of the corresponding inverse operations (for more details, refer :cite:p:`wille2019towardsHDLsynthesis`). The figure below depicts the circuit which results when synthesizing the program from above using the line-aware scheme.
+    This synthesis scheme aims to the keep the number of additional circuit lines as small as possible. To achieve this, the corresponding intermediate results of each operation are computed and stored using one of the available circuit lines (rather than additional circuit lines). Afterwards, the inputs are re-computed with the help of the corresponding inverse operations. The figure below depicts the circuit which results when synthesizing the program from above using the line-aware scheme.
```

Comment:
These changes should be reverted as well. The papers listed before provide a way more detailed overview of the corresponding methods and it is no use to insert an additional layer of indirection by just referring to the newly published journal here.